### PR TITLE
Misleading prompts

### PR DESCRIPTION
**Pull request by Dastardi**
Small addition to prompt reliability intro and accompanying bibliography update

### CODE REVIEW

File: docs/reliability/intro.md [hunk 1]
```diff
@@ -11,13 +11,14 @@ To a certain extent, most
 of the previous techniques covered have to do with improving completion
 accuracy, and thus reliability, in particular self-consistency(@wang2022selfconsistency).
 However, there are a number of other techniques that can be used to improve reliability,
-beyond basic prompting strategies.
+beyond basic prompting strategies. 
 
-%%LLMs|LLM%% exhibit various problems including hallucinations(@ye2022unreliability), 
+While %%LLMs|LLM%% display a high level of charitable reliability even in the face of actively misleading prompts(@webson2023itscomplicated),
```
Comment (trigaten) [inline docs/reliability/intro.md:16]: What is charitable reliability?

Comment (Dastardi) [reply docs/reliability/intro.md:16]: LLMs reliably giving a prompt a charitable interpretation - essentially, the model can work its way around misguiding or incorrect data (e.g. misspelled words or bad grammar, or even fully tangential information in the same prompt) and understand what you're *trying* to say.

Comment (trigaten) [reply docs/reliability/intro.md:16]: Ah, can you define this here? People will not know it.

Comment (Dastardi) [reply docs/reliability/intro.md:16]: Yep, sure thing!

**Pull request merged**